src/Evolution/Systems/ScalarWave/BoundaryConditions/ConstraintPreservingSphericalRadiation.hpp

```diff
@@ -139,6 +139,9 @@ convert_constraint_preserving_spherical_radiation_type_from_yaml(
  *     \partial_r^2+\frac{4}{r}\partial_t
  *     +\frac{4}{r}\partial_r + \frac{2}{r^2}\right)\Psi \notag \\
  *     &=n^i(\partial_t\Phi_i-\partial_i\Pi) + n^i n^j\partial_i\Phi_j -
+ *     \frac{4}{r}\Pi+\frac{4}{r}n^i\Phi_i + \frac{2}{r^2}\Psi \notag \\
```

Comment:
Could there be a comment here that says what frame `partial_t` means?  (doesn't hurt to repeat info to make it easier to understand frames)